### PR TITLE
:sparkles: Add metrics info card

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206536111880008

### CODE REVIEW

**Comment by git-marcopitra** [comment]
image
Why ur values are not working?

**Comment by git-marcopitra** [comment]
image

Why is it displayed like this?

File: views/metrics/index.tsx [hunk 1]
```diff
@@ -22,10 +23,14 @@ const Metrics: FC = () => (
       Metrics
     </Typography>
     <Box
-      variant="container"
+      display="flex"
```
Comment (git-marcopitra) [inline views/metrics/index.tsx:26]: remove these new changes

File: views/metrics/info-card/info-card.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { InfoCard, Typography } from '@interest-protocol/ui-kit';
```
Comment (git-marcopitra) [inline views/metrics/info-card/info-card.tsx:1]: Bad naming

File: views/metrics/info-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { FC, useEffect, useState } from 'react';
+import { v4 } from 'uuid';
+
+import { getMetric } from '@/api/metrics';
+import { formatDollars, formatMoney } from '@/utils';
+
+import { TOP_INFO_CARDS_DATA } from '../metrics.data';
+import InfoCards from './info-card';
+
+const InfoCardsList: FC = () => {
```
Comment (git-marcopitra) [inline views/metrics/info-card/index.tsx:11]: Overview, file name as well

File: views/metrics/info-card/info-card.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import { FC } from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+import { InfoCardProps } from './info-card.types';
+
+const InfoCards: FC<InfoCardProps> = ({
```
Comment (git-marcopitra) [inline views/metrics/info-card/info-card.tsx:7]: InfoCard > filename ..../`overview/info-card.tsx`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Comment by MarioBatalha** [comment]
> image
> 
> Why is it displayed like this?

It's not part of this task

File: components/svg/chart.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    style={{ maxHeight, maxWidth }}
+    viewBox="0 0 24 24"
+    fill="none"
+    {...props}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M13.9444 5.25L13.3889 5.80556V8.58333H10.0556L9.5 9.13889V11.9167H6.16667L5.61111 12.4722V16.9167H4.5V18.0278H6.16667H10.0556H13.9444H17.8333H19.5V16.9167H18.3889V5.80556L17.8333 5.25H13.9444ZM17.2778 16.9167V6.36111H14.5V9.13889V16.9167H17.2778ZM13.3889 16.9167V9.69444H10.6111V12.4722V16.9167H13.3889ZM6.72222 13.0278H9.5V16.9167H6.72222V13.0278Z"
+      fill="#1B1B1F"
```
Comment (KipandaJr) [inline components/svg/chart.tsx:16]: CurrentColor here

File: views/metrics/index.tsx [hunk 6]
```diff
@@ -23,9 +24,14 @@ const Metrics: FC = () => (
     </Typography>
     <Box
       variant="container"
+      display="flex"
+      flexDirection="column"
+      justifyContent="center"
+      alignContent="center"
```
Comment (KipandaJr) [inline views/metrics/index.tsx:30]: Do u need this changes? i think its break our responsiveness.

File: views/metrics/info-cards/index.tsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+import { getMetric } from '@/api/metrics';
+import { formatDollars, formatMoney } from '@/utils';
+
+import { INFO_CARDS_DATA } from '../metrics.data';
+import Overview from './overview';
+
+const InfoCards: FC = () => {
+  const [data, setData] = useState<[]>();
+
+  console.log('get overview data', data);
```
Comment (KipandaJr) [inline views/metrics/info-cards/index.tsx:14]: remove this line

File: views/metrics/info-cards/index.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+          key={v4()}
+          Icon={Icon}
+          description={description}
+          amount={(money ? formatDollars : formatMoney)(
+            data?.[index] ?? 0
+          ).toString()}
+          loading={!data?.length}
+        />
+      ))}
+    </Box>
```
Comment (KipandaJr) [inline views/metrics/info-cards/index.tsx:38]: Did u test it on mobile version?

File: views/metrics/info-cards/overview/index.tsx [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+  Icon,
+  description,
+  amount,
+  loading,
+}) => (
+  <>
+    <InfoCard
+      title={description}
+      Icon={<Icon maxWidth="1.5rem" maxHeight="1.5rem" width="100%" />}
+      width={['32rem', '32rem', '32rem', '100%']}
```
Comment (KipandaJr) [inline views/metrics/info-cards/overview/index.tsx:17]: this size on modal will break ur responsiveness

File: views/metrics/info-cards/index.tsx [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+      display={['flex', 'flex', 'flex', 'grid']}
+      flexDirection={['column', 'row', 'row', 'row']}
+      gridTemplateColumns="repeat(3, 1fr)"
+    >
+      {INFO_CARDS_DATA.map(({ Icon, description, money }, index) => (
+        <Overview
+          key={v4()}
+          Icon={Icon}
+          description={description}
+          amount={(money ? formatDollars : formatMoney)(
```
Comment (KipandaJr) [inline views/metrics/info-cards/index.tsx:32]: Number of POOLS is not money or decimal value like `50.00` its only a int number `50`

File: views/metrics/info-cards/index.tsx [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+  useEffect(() => {
+    getMetric('get-overview').then(setData);
+  }, []);
+  return (
+    <Box
+      width="100%"
+      gap="s"
+      display={['flex', 'flex', 'flex', 'grid']}
+      flexDirection={['column', 'row', 'row', 'row']}
```
Comment (KipandaJr) [inline views/metrics/info-cards/index.tsx:24]: its correct this values?

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
its break in this Breakpoint `1039`

File: views/metrics/info-cards/index.tsx (hunk 8 above)
Comment (MarioBatalha) [reply views/metrics/info-cards/index.tsx:38]: Yes indeed

File: views/metrics/index.tsx [hunk 12]
```diff
@@ -23,13 +24,22 @@ const Metrics: FC = () => (
     </Typography>
     <Box
       variant="container"
-      gap={['xs', 'xs', 'xs']}
-      gridTemplateColumns={['1fr', '1fr', '1fr', '1fr 1fr']}
+      gap={['xs', 'xs', 'xs', 'xs']}
```
Comment (KipandaJr) [inline views/metrics/index.tsx:27]: only `gap="xs"` its enought

File: views/metrics/index.tsx [hunk 13]
```diff
@@ -23,13 +24,22 @@ const Metrics: FC = () => (
...
+      gridTemplateColumns={['1fr', '1fr', '1fr', '1fr']}
```
Comment (KipandaJr) [inline views/metrics/index.tsx:28]: only `gridTemplateColumns="1fr"`

File: views/metrics/index.tsx [hunk 14]
```diff
@@ -23,13 +24,22 @@ const Metrics: FC = () => (
...
       <TotalLiquidity />
       <DailyVolume />
-      <ActiveWallets />
-      <TVLPools />
+      <Box
+        width="100%"
+        display="flex"
+        flexDirection={['column', 'column', 'row', 'row']}
+        justifyContent="center"
+        gap={['xs', 'xs', 'xs']}
```
Comment (KipandaJr) [inline views/metrics/index.tsx:38]: only `gap="xs"` its enought

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: views/metrics/index.tsx (hunk 14 above)
Comment (MarioBatalha) [reply views/metrics/index.tsx:38]: If I remove this value the layout will break, cuz I'm forcing to keep the same size for all break point.

Comment (KipandaJr) [reply views/metrics/index.tsx:38]: not make sense this affirmation

**Review by git-marcopitra — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Pull request merged**